kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/util/TaskQueryFilterSpecBuilderTest.java

```diff
@@ -0,0 +1,16 @@
+package org.kie.server.api.util;
+
+import org.junit.Test;
+
+public class TaskQueryFilterSpecBuilderTest {
```

Comment:
@domhanak Good question, don't know :-) Probably a placeholder that I never implemented ....